apps/crn-server/src/validation/compliance-report.validation.ts

```diff
@@ -16,8 +16,11 @@ const complianceReportPostRequestValidationSchema: JSONSchemaType<ComplianceRepo
       discussionId: { type: 'string', nullable: true },
       versionId: { type: 'string', nullable: true },
       manuscriptId: { type: 'string', nullable: true },
```

Comment:
Hey, I was wondering if this `manuscriptId` is ever nullable, is it? if not maybe we can have `nullable: false` and then in the route file we could have

```
    const manuscript = await manuscriptController.fetchById(manuscriptId);
```

instead of 

```
    const manuscript =
      manuscriptId && (await manuscriptController.fetchById(manuscriptId));
```